modulefiles/gsi_common.lua

```diff
@@ -30,5 +31,7 @@ load(pathJoin("nemsio", nemsio_ver))
 load(pathJoin("wrf_io", wrf_io_ver))
 load(pathJoin("ncio", ncio_ver))
 load(pathJoin("crtm", crtm_ver))
-load(pathJoin("ncdiag",ncdiag_ver))
+--load(pathJoin("ncdiag",ncdiag_ver))
```

Comment:
This could to be uncommented after WCOSS2 makes updates. 
All other systems have it installed, including Orion.